app/views/orders/show.html.erb

```diff
@@ -0,0 +1,33 @@
+<h1>Order Details</h1>
+<h4> Placed At: <%= @order.created_at %> </h2>
+
+<table class="table">
+  <th>Order Status</th>
+  <th>Order Placed At</th>
+
+  <tr>
+    <td><%= @order.status %></td>
+    <td><%= @order.get_relevant_time %></td>
```

Comment:
i think we'll want to show the created at time in addition to the the cancelled or completed time